src/components/templates/Entrepreneurship/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { Box, Grid } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import axios from "axios";
+import React, { ReactElement } from "react";
+import { useNavigate } from "react-router-dom";
+import { theme } from "../../../themes";
+import TypographyText from "../../atoms/Typography";
+import BookCard from "../../organisms/BookCard";
+
+const useStyles = makeStyles({
+  bookCardStyles: {
+    "&:hover": {
+      backgroundColor: theme.palette.backgroundcolor.main,
+      cursor: "pointer",
+    },
+  },
+});
+
+interface ICard {
+  id: number;
+  title: string;
+  category: string;
+  author: string;
+  time: string;
+  reads: string;
+  imageSrc: string;
+  isAddedToLib: boolean;
+  isFinishedReading: boolean;
+}
+
+type EntrepreneurshipTemplateProps = {
+  header: ReactElement<any>;
+  footer: ReactElement<any>;
+  cardBanner: ReactElement<any>;
+  searchBar: ReactElement<any>;
+  blinks: ICard[];
+};
+
+const EntrepreneurshipTemplate: React.FC<EntrepreneurshipTemplateProps> = (
+  props
+) => {
+  const styleClasses = useStyles();
+  const navigate = useNavigate();
+  const handleClick = async (data: ICard) => {
+    if (!data.isAddedToLib) {
+      await axios
+        .patch(`http://localhost:3000/cardsInfo/${data.id}`, {
+          isAddedToLib: true,
+        })
+        .then((response: any) => {
+          let path = `/book/${data.id}`;
+          navigate(path);
+        });
+    } else {
+      let path = `/book/${data.id}`;
```

Comment:
same here..you can initialize path variable using let before "if" and then change the value.